Type Check failure on CI

```
Error: app/models/user.server.ts(13,14): error TS2339: Property 'name' does not exist on type 'User'.
Error: app/models/user.server.ts(19,7): error TS2322: Type '{ username: string; name: any; email: string; }' is not assignable to type '(Without<UserCreateInput, UserUncheckedCreateInput> & UserUncheckedCreateInput) | (Without<...> & UserCreateInput)'.
  Object literal may only specify known properties, and 'name' does not exist in type '(Without<UserCreateInput, UserUncheckedCreateInput> & UserUncheckedCreateInput) | (Without<...> & UserCreateInput)'.
Error: prisma/seed.ts(19,7): error TS2322: Type '{ username: string; name: string; email: string; }' is not assignable to type '(Without<UserCreateInput, UserUncheckedCreateInput> & UserUncheckedCreateInput) | (Without<...> & UserCreateInput)'.
  Object literal may only specify known properties, and 'name' does not exist in type '(Without<UserCreateInput, UserUncheckedCreateInput> & UserUncheckedCreateInput) | (Without<...> & UserCreateInput)'.
Error: Process completed with exit code 2.
```

--- a/app/models/user.server.ts
+++ b/app/models/user.server.ts
@@ -10,13 +10,15 @@ export async function getUserByUsername(username: User['username']) {
 
 export async function createUser(
   username: User['username'],
-  name: User['name'],
+  firstName: User['firstName'],
+  lastName: User['lastName'],
   email: User['email']
 ) {
   return prisma.user.create({
     data: {
       username,
-      name,
+      firstName,
+      lastName,
       email,
     },
   });

--- a/prisma/seed.ts
+++ b/prisma/seed.ts
@@ -5,7 +5,8 @@ const prisma = new PrismaClient();
 
 async function seed() {
   const username = 'starchartdev';
-  const name = 'Starchart Developer';
+  const firstName = 'Starchart';
+  const lastName = 'Developer';
   const email = 'dev@starchart.invalid';
 
   // cleanup the existing database
@@ -16,7 +17,8 @@ async function seed() {
   await prisma.user.create({
     data: {
       username,
-      name,
+      firstName,
+      lastName,
       email,
     },
   });

